packages/friends/yml/en/mary-dudley.yml

```diff
@@ -19,12 +20,13 @@ documents:
     description:
       Mary Dudley (1750-1823) was a seeker of truth and righteousness from her earliest
       days. When around 20 years old, she joined the Methodists for two or three years,
-      and was much esteemed by John Wesley for her humble devotion and pious example. She
-      soon found, however, that their active and outward zeal was not compatible with the
-      spiritual poverty she felt in herself, and the great need she had seen for total
-      dependence upon the Spirit of God for all true worship and ministry. Finding a home
-      among the Quakers, she grew to become a powerful minister and a "mother in Israel",
-      pointing many to the covenant of light and life in Jesus Christ.
+      and was much esteemed by John Wesley for her humble devotion and pious example.
+      Before long, however, she began to feel that their active and outward zeal was not
+      compatible with the spiritual poverty she felt in herself, and the great need she
+      had seen for total dependence upon the Spirit of God for all true worship and
+      ministry. Finding a home among the Quakers, she grew to become a powerful minister
+      and a "mother in Israel", traveling often, and pointing many hearts to the covenant
```

Comment:
I think this should be `"mother in Israel,"` -- moving the comma within the quotes.